render-wasm/src/shapes.rs

```diff
@@ -339,10 +388,36 @@ impl Shape {
         self.hidden
     }
 
-    pub fn bounds(&self) -> math::Rect {
+    // TODO: Maybe store this inside the shape
```

Comment:
Are we leaving the TODO here?

Reply:
Yes, why not? I think it's a good idea to keep todos for possible future improvements even if we don't do them now.